### PR TITLE
Frontend merge into main

### PR DESCRIPTION
**Pull request by Ajla115**

### CODE REVIEW

File: src/api/movieApi.ts [hunk 1]
```diff
@@ -0,0 +1,72 @@
+import axios from "axios";
+import { Movie, AddMovieDTO } from "../types/Movie";
+
+
+const API_URL = "http://localhost:8080/api/movies";
```
Comment (ibecir) [inline src/api/movieApi.ts:5]: Make the URL as a global constant

File: src/api/userApi.ts [hunk 2]
```diff
@@ -0,0 +1,58 @@
+import { useState, useEffect } from "react";
+import { useMutation } from "@tanstack/react-query";
+import axios from "axios";
+
+const API_URL = "http://localhost:8080/api/users";
+
+export const useNotificationToggle = () => {
```
Comment (ibecir) [inline src/api/userApi.ts:7]: Should be a global constant

File: src/components/EmailInput.tsx
Comment (ibecir) [inline src/components/EmailInput.tsx:1]: Comments should be removed

File: src/index.tsx
Comment (ibecir) [inline src/index.tsx:1]: Comments should be removed

**Review by ibecir — CHANGES_REQUESTED** [review]
Code looks solid, please remove unnecessary files and REMOVE ALL OF THE COMMENTS THAT ARE CODE LEFTOVERS. After providing the fixes open a new PR.

**Comment by Ajla115** [comment]
Made necessary changes - removed unused files, deleted comments and made a global api constant

**Pull request merged**